vpr/src/route/rr_graph_area.cpp

```diff
@@ -167,7 +167,7 @@ void count_bidir_routing_transistors(int num_switch, int wire_to_ipin_switch, fl
                 num_edges = device_ctx.rr_nodes[from_node].num_edges();
 
                 for (iedge = 0; iedge < num_edges; iedge++) {
-                    RRNodeId to_node = RRNodeId(device_ctx.rr_nodes[from_node].edge_sink_node(iedge));
+                    RRNodeId to_node = RRNodeId(size_t(rr_graph.edge_sink_node(RRNodeId(from_node), iedge)));
```

Comment:
I think the return type of ``edge_sink_node()`` is ``RRNodeId``. You can avoid the type cast ``RRNodeId(size_t())`` here?